sdk/core/test/unit/services/WalletStorageService.test.ts

```diff
@@ -149,6 +171,22 @@ describe('WalletStorageService', () => {
     })
   })
 
+  describe('#getAllCredentialsWithBadEncryption', () => {
+    it('should filter bad credential from Affinidi Vault without throwing error', async () => {
+      sinon
+        .stub(AffinidiVaultApiService.prototype, 'searchCredentials')
+        .withArgs(region)
+        .callsFake(function anon(): any {
+          return { body: { credentials: [encryptedBadCredential] } }
```

Comment:
WDYT to extend this list of credentials with good ones?
E.g. Good {id=1} Bad {id=2} Good {id=3} Bad {id=4} Good {id=5}
and then compare the result contains only `id in 1, 3, 5`

Reply:
the limitation is - i can only create walletStorageService with either `createWalletStorageService` or `createWalletStorageServiceWithBadEncryption`. The main goal of the test is to return array without throwing exception. 

Yes, test can be extended but I prefer to find the actual root cause and fix it. By right, no credential should be corrupted and fail to decrypt (since same sdk is used to encrypt)